src/Service/ElasticsearchService.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/**
+ * @file
+ * Contains service to integrate with elasticsearch.
+ */
+
+namespace App\Service;
+
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * Class ElasticsearchService.
+ */
+class ElasticsearchService implements ElasticsearchServiceInterface
+{
+    private const SEARCH_SIZE = 100;
+
+    private $elasticsearchURL;
+    private $httpClient;
+
+    /**
+     * StatisticsExtractionService constructor.
+     *
+     * @param \Symfony\Contracts\HttpClient\HttpClientInterface $httpClient
+     *   The http client
+     * @param $boundElasticsearchURL
+     *   Url of elasticsearch instance
+     */
+    public function __construct(HttpClientInterface $httpClient, $boundElasticsearchURL)
```

Comment:
Missing type for `$boundElasticsearchURL`.

Reply:
Fixed